### PR TITLE
 nav bar admin

### PR DESCRIPTION
**Pull request by endricklosange**

### CODE REVIEW

**Review by maxime-guichard — APPROVED** [review]

**Review by Erdnaxela45 — DISMISSED** [review]

**Review by Erdnaxela45 — DISMISSED** [review]

File: templates/_navbarAdmin.html.twig [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5 active" href="#">Etudiant</a>
+            </li>
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5 active" href="#">Entreprise</a>
+            </li>
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5 active" href="#">Connexion</a>
+            </li>
+        </ul>
```
Comment (LucyMerlier) [inline templates/_navbarAdmin.html.twig:22]: indentation

File: templates/_navbarAdmin.html.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<nav class="navbar navbar-expand-sm bg-primary navbar-dark">
+    <a class="navbar-brand" href="/">
+    <img src="logo.jpg" alt="logo" class"navbar-logo">
+  </a>
```
Comment (LucyMerlier) [inline templates/_navbarAdmin.html.twig:4]: indentation

File: templates/_navbarAdmin.html.twig [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    <div class="collapse navbar-collapse justify-content-around" id="navbarNavAltMarkup">
+        <ul class="navbar-nav navlinks">
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5 active" href="/">Accueil</a>
+            </li>
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5 active" href="#">Offre d'emploi</a>
```
Comment (LucyMerlier) [inline templates/_navbarAdmin.html.twig:11]: OffreS

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments + names of templates should be in snake_case

**Review by Wowlfy — APPROVED** [review]

**Review by JuMn88 — APPROVED** [review]

**Review by maxime-guichard — DISMISSED** [review]

**Review by Wowlfy — APPROVED** [review]

**Review by Wowlfy — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**